### ISSUE TITLE
Training Loss Function prints 0

### ISSUE THREAD

**Issue by renespeck** [dbpedia/neural-rdf-verbalizer#9]
Hi thanks for the code. 

I successfully ran the application with the default configuration 'gat'. 

I tried to run the application with a 'transformer' configuration, so encoder and decoder are transformer. 
The applications starts and runs but the Training Loss Function always prints 0. (I tried several parameters in the configurations but nothing changed)

I'm not sure what happens, is my data wrong, my configuration  wrong or is it maybe a bug? 
My training data, are files of lines with sentences. 

So do you have a example configuration and input data so run a 'transformer' based network? 

Any help is highly appreciated.  

Thank you.

**Comment by DwaraknathT** [comment]
Could you please provide some log files or error stack to trace ?

**Comment by renespeck** [comment]
Hi sure, the link to the log shows the preprocess first and then the training log at line 80. My environment list starts at line 5432. 

https://gist.github.com/renespeck/c6a667883cf6601d61608d07e1d337fe
